tests/assets/objects/test-ally.json

```diff
@@ -9,26 +9,32 @@
     {
       "name": "Transform",
       "data": {
-        "position": {
-          "x": 0.0,
-          "y": 0.0,
-          "z": 0.0
-        },
-        "rotation": {
-          "x": 0.0,
-          "y": 0.0,
-          "z": 0.0
-        },
-        "scale": {
-          "x": 1.0,
-          "y": 1.0,
-          "z": 1.0
+        "invisible": {
```

Comment:
Why did you add invisible ?

Reply:
Field are groupped by their field group and it reflects in the json from now on